cypress/integration/login.js

```diff
@@ -17,22 +9,45 @@ describe('Sourcebooks login', function() {
     ];
 
     function makeid(length) {
-        var result = '';
-        var characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
-        var charactersLength = characters.length;
-        for (var i = 0; i < length; i++) {
-           result += characters.charAt(Math.floor(Math.random() * charactersLength));
-        }
-        return result;
+        return 'Random_Name_' + Math.floor(Math.random() * 1000000000000);
     }
 
-    it('Should be able to create new task as Admin', function () {
+    it('Should display validation for empty user after attempted loggin', function () {
+        
+        cy.visit('/');
+        cy.get('.Select.not-valid').should('not.visible')
+        cy.get('[type="submit"]').click();
+        cy.get('.Select.not-valid').should('be.visible')
+    })
+
+    function testLogIn(role) {

```

Comment:
It would be best to keep functions separated from test cases or at least not mix them inside describe